shippable.yml

```diff
@@ -8,21 +8,51 @@ matrix:
   exclude:
     - env: T=none
   include:
-    - env: T=aix/7.2/1
-    - env: T=osx/10.11/1
-    - env: T=rhel/7.6/1
-    - env: T=rhel/8.1/1
-    - env: T=freebsd/11.1/1
-    - env: T=freebsd/12.1/1
-    - env: T=linux/centos6/1
-    - env: T=linux/centos7/1
-    - env: T=linux/centos8/1
-    - env: T=linux/fedora30/1
-    - env: T=linux/fedora31/1
-    - env: T=linux/opensuse15py2/1
-    - env: T=linux/opensuse15/1
-    - env: T=linux/ubuntu1604/1
-    - env: T=linux/ubuntu1804/1
+    - env: T=2.9/aix/7.2/1
```

Comment:
What's happened to `sanity` and `units`?

Reply:
@gundalow thos are running in Zuul now, we're aiming to move everything to Zuul before Shippable is sunset by JFrog but still leveraging Shippable for now for a lot of stuff until we have time to transition.

Reply:
Cool, thanks.